sections/php.zsh

```diff
@@ -27,7 +27,9 @@ spaceship_php() {
 
   spaceship::exists php || return
 
-  local php_version=$(php -v 2>&1 | grep --color=never -oe "^PHP\s*[0-9.]\+" | awk '{print $2}')
+  # 1. Split output to array on space and return first element
+  # 2. Split output from step 1 on (-) and return second element
+  local php_version=${${(@s|-|)$(php -v 2>&1)[2]}[1]}
```

Comment:
This doesn't work for me.
`PHP 7.2.11 (cli) (built: Oct 11 2018 16:23:09) ( NTS )`
will print `7` instead of `7.2.11`

Reply:
> `PHP 7.2.11 (cli) (built: Oct 11 2018 16:23:09) ( NTS )`
will print `7` instead of `7.2.11`

This is confusing, So `php -v` doesn't have stable interface for CLI version and returns different values on different platforms ? 

> Deleting `[1]` is OK.

Dropping `[1]` would also include `extra` section information from  `major.minor.release[extra]`

```zsh
~/tests/spaceship via 🐘 v7.2.10 0ubuntu0.18.04.1
[I] ➜
```

Reply:
> `PHP 7.2.11 (cli) (built: Oct 11 2018 16:23:09) ( NTS )`
will print `7` instead of `7.2.11`

This is confusing, Seems like `php -v` doesn't have stable interface for CLI version and returns different values on different platforms !! If that's the case, We could revert the case until better solution. 

> Deleting `[1]` is OK.

Dropping `[1]` would also include `extra` section information from  `major.minor.release[extra]`

```zsh
~/tests/spaceship via 🐘 v7.2.10 0ubuntu0.18.04.1
[I] ➜
```

Reply:
Changes to `php` section reverted.